src/go/k8s/apis/redpanda/v1alpha1/console_types.go

```diff
@@ -134,6 +134,10 @@ type Server struct {
 // Schema defines configurable fields for Schema Registry
 type Schema struct {
 	Enabled bool `json:"enabled"`
+
+	// Indication on whether to use the node CA as trust when connecting to the schema registry.
+	// If not set, the public CAs will be used.
+	UseNodeCA bool `json:"useNodeCA,omitempty"`
```

Comment:
kubernetes nodes don't have their own CA

Reply:
This is not the actual k8s node CA. I was trying to use the term similar to `[NodeSecretRef](https://github.com/redpanda-data/redpanda/blob/2434e021f8685ffce55c99eba7bc3ef4dff790be/src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go#LL804C2-L804C15)`. I know it is confusing. I will rename to `UseSchemaRegistryCA`.
 It is used to configure Console to trust the CA issuing the schema registry certificate rather than the public well-known trusted CAs. In the e2e test, the schema registry certificate is self signed, this property will allow Console to trust the self signed cert.